sdk/maps/maps-search/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/maps-search"
+      ServiceDirectory: maps
+      SupportedClouds: 'Public,Canary'
+      EnvVars:
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_LOG_LEVEL: info
```

Comment:
Did you mean to add AZURE_LOG_LEVEL?

Reply:
I think we don't need this. Thanks for the reminder!